openedx/features/caliper_tracking/transformers/team_transformers.py

```diff
@@ -153,3 +153,82 @@ def edx_team_changed(current_event, caliper_event):
     })
 
     return caliper_event
+
+
+def edx_team_learner_removed(current_event, caliper_event):
+    """
+    When a user leaves a team or is removed by someone else,
+    the server emits an edx.team.learner_deleted event.
+
+    This event is also triggered when a team is deleted,
+    because all members are removed when a team is deleted.
+
+    Course team members who have any of the Staff, Admin, Discussion Admin,
+    Discussion Moderator, or Community TA roles can remove learners from teams.
+
+    :param current_event: default event log generated.
+    :param caliper_event: caliper_event log having some basic attributes.
+    :return: updated caliper_event.
+    """
+    username = utils.get_username_from_user_id(current_event['event']['user_id'])
+
+    user_link = '{lms_url}{profile_link}'.format(
```

Comment:
Shouldn't this code be moved into a separate util method by now?

Reply:
Done